app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```

Comment:
1. `page_title = ""` không cần khởi tạo giá trị mặc định cho `page_title` em nhé

2. `"Ruby on Rails Tutorial Sample App"`
Tìm hiểu về I18n nào, và sử dụng I18n cho trường hợp bên trên, và trong các file html.erb nhé 

3. đoạn `if -else` em dùng toán tử ngôi số 3 nhé
`condition ? result_for_true : result_for_false`

4. Phân biệt cho chị `empty?` `blank?` và `nil?` nhé
Ở đây thì em dùng `blank?` thay vì `empty?` nhé em